rollup/internal/controller/blob_uploader/blob_uploader.go

```diff
@@ -242,8 +242,8 @@ func (b *BlobUploader) GetFirstUnuploadedBatchByPlatform(ctx context.Context, st
 		break
 	}
 
-	// disable this check to upload blobs before it's committed. This is to 
```

Comment:
I'm using an extension called `Trailing Spaces` which highlight such characters in your IDE, it's quite useful.